recipes/daggy/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+  "2.1.1":
+    folder: 2.1
```

```suggestion
    folder: "2.1"
```

That was a float (not a string), which translated to a folder... well, it was raising a weird error in the internals. I'm adding a quick fix, but it will take some time until it gets to production.